README.md

```diff
@@ -1,84 +1,63 @@
-
 # ![alt text](https://cdn.discordapp.com/attachments/1160692058017763581/1175551711281168456/LMP-64.png?ex=656ba4d8&is=65592fd8&hm=ccaf321556119a09510514835ccadded74ee8bcc02fb48ec5697d5e9defcdd22&) Lampray 
-A Mod Manager for games, made for Linux! As an alternative to Vortex and MO2.
-
-This tool is in its early days please see the contributing section to help add more to it! 
-Big thanks to airtonix & SnazzyPanda for their amazing work on Lampray.
-
-### Contributors discussion / Early Peeks
-If you want early information about lamp, and how things are progressing join me on my very basic development discord.
-[discord](https://discord.gg/5macMedevy)
 
+Lampray is a mod manager for gaming on Linux! If you'd like to help improve Lampray, you can:
 
-## List of currently supported games
-- Baldur's Gate 3
-- Cyberpunk 2077
+- [Report an issue](https://github.com/CHollingworth/Lampray/issues/new?assignees=&labels=bug&projects=&template=-game--bug-report.md&title=)
+- [Request a feature](https://github.com/CHollingworth/Lampray/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=)
+- [Join the Discord](https://discord.gg/5macMedevy)
+- [Contribute](./CONTRIBUTING.md)
 
-## List of planned support
-- [ ] Skyrim (SE & AE)
-- [ ] Fallout 4
-- [ ] Oblivion
-- [ ] Morrowind
-- [ ] The Sims 4
+If you'd like to learn more about Lampray, see [Lampray Docs](./docs/lampray-docs.md).
 
-## List of currently supported archives
-- zip
-- 7z
-- rar (No longer supported)
+## Quick start
```

Comment:
Update **Quick start** to cover setting up, building, and launching Lampray _without_ building from source. Those steps are separate.

Reply:
For non-build method, simply download executable, then give it execute permissions (`chmod +x`) then run file.

Reply:
Note: First time Lampray runs, it'll build the following in the _same_ directory:

- `Lamp_Data`
- `imgui.ini`
- `lamp.log`
- `Lampray` (the executable)